frontend/src/pages/FaqPage.js

```diff
@@ -44,75 +45,149 @@ export default function FaqPage({ user }) {
 const faqData = [
 	{
 		title: "What is an NFT?",
-		text: ["Non-fungible token or NFT is a type of crypto asset that represents digital works of art and other collectibles into one-of-a-kind. Each NFT is unique and exists in a single copy, it cannot be divided, and all information about its author, buyer and all transactions with it is securely stored in the blockchain. In other words, an NFT is a digital certificate attached to a unique object."]
+		text: [
+			"A non-fungible token or NFT for short is a type of digital asset. " +
+			"It can represent, for example, digital works of art and other collectibles, each NFT being one-of-a-kind, i.e. \"non-fungible\". " +
+			"Each NFT has a unique value. It's indivisible and all meta information about its author, buyer and associated transactions is immutably and publicly stored on blockchain.",
+
+			"An NFT can be thought of like a digital certificate of market rights attached to a digital artificact.",
+
+			"Yet, in most parts of the world, this meaning relies on community conventions and metaverse applications and not actual legal rights.",
+
+			"NFTs are not a copy protection for digital content."
+		]
 	},
 	{
-		title: "Which blockchains does NFT world! support?",
-		text: ["The NFT World! currently supports Ethereum blockchain."]
+		title: "Which blockchains does NFT The World! support and why?",
+		text: [
+			"NFT The World! only supports Ethereum mainly because of popularity and a big community of users and developers."
```

Comment:
```suggestion
			"NFT The World! only supports the Ethereum blockchain mainly because of popularity and a big community of users and developers."
```